test/pylib/manager_client.py

```diff
@@ -152,3 +160,11 @@ async def server_update_config(self, server_id: str, key: str, value: object) ->
                                        {"key": key, "value": value})
         if resp.status != 200:
             raise Exception(await resp.text())
+
+    async def get_server_uuid(self, server_id: str) -> None:
+        """Get server UUID through Scylla REST API"""
+        return await self.api.get_server_uuid(server_id)
+
+    async def server_wait_known(self, server_id: str, expected_server_id: str) -> bool:
```

Comment:
This function is not used in this PR.
Please get rid of it and `wait_for_host_known` to make the PR easier to review.